src/components/Alert.ts

```diff
@@ -1,6 +1,6 @@
-import { DOM, StatelessComponent } from "react";
+import { StatelessComponent, createElement } from "react";
 
 export const Alert: StatelessComponent<{ message?: string }> = ({ message }) =>
```

Comment:
Make alert generic so that it can be used for different kind of alert like success, info, etc.